app/build.gradle

```diff
@@ -5,7 +5,7 @@ apply from: '../gitutils.gradle'
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
-apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-parcelize'
```

Comment:
'kotlin-android-extensions' is deprecated so we're removing it in place of view binding instead.

We use parcelize, which is part of the old extensions plugin. It's now been moved out to its own plugin, hence we're adding that one back in. From from https://developer.android.com/topic/libraries/view-binding/migration:

> Note: If your app uses Parcelable features, switch to using the standalone kotlin-parcelize Gradle plugin described in [Parcelable implementation generator](https://developer.android.com/kotlin/parcelize).